library/std/src/sys/vxworks/ext/fs.rs

```diff
@@ -9,7 +9,7 @@ use crate::sys_common::{AsInner, AsInnerMut, FromInner};
 
 /// Unix-specific extensions to [`File`].
 ///
-/// [`File`]: ../../../../std/fs/struct.File.html
+/// [`File`]: crate::fs::File
```

Comment:
`fs` is in scope: https://github.com/rust-lang/rust/blob/9463380b0ceeca6c99d7466f90f1bf9fe4732d8d/library/std/src/sys/vxworks/ext/fs.rs#L3

```suggestion
/// [`File`]: fs::File
```